flink-python/src/main/java/org/apache/flink/streaming/api/runners/python/beam/BeamPythonFunctionRunner.java

```diff
@@ -190,7 +194,8 @@ public BeamPythonFunctionRunner(
             MemoryManager memoryManager,
             double managedMemoryFraction,
             FlinkFnApi.CoderInfoDescriptor inputCoderDescriptor,
-            FlinkFnApi.CoderInfoDescriptor outputCoderDescriptor) {
+            FlinkFnApi.CoderInfoDescriptor outputCoderDescriptor,
+            @Nullable Map<String, FlinkFnApi.CoderInfoDescriptor> sideOutputCoderDescriptors) {
```

Comment:
Remove the @Nullable annotation and use Preconditions.checkNotNull to make sure that given value is not null.

Reply:
Table operators don't have side outputs, so I assume it's reasonable to use `null` for `sideOutputCoderDescriptors` in `BeamTablePythonFunctionRunner`?